add new handlers

nw13slx opened the pull request:
The old default list seems to miss two error handlers from Custodian 2022.5.26

nw13slx committed:
add new handlers

nw13slx committed:
add new handlers

astamminger commented:
Hi @nw13slx! Thanks for adding the missing handlers :) It seems you added the changes to the master branch directly. Could you please create a new branch originating from develop and cherry-pick your changes into that branch instead? Then create a new pull request from that branch with ``develop`` as target branch, thanks!

//Edit: Don't mind the failing tests. I'll fix them this afternoon

nw13slx committed:
Merge branch 'aiida-cusp:develop' into develop

Commit:
add space

Commit:
Merge branch 'master' of github.com:nw13slx/aiida-cusp

astamminger committed:
Add missing default to IncorrectSmearingHandler

IncorrectSmearingHandler requires the output file of a VASP calculation to be instantiated. This commit adds the 'vasprun.xml' as the default output file that will be passed to the IncorrectSmearingHandler

astamminger committed:
Constrain Custodian to versions >=2021.1.7

The newly added Handlers are only available for Custodian versions equal to or above v2021.1.7. This commit updates the required version in the projects `setup.json` to enforce this constraint

astamminger commented:
Thanks a lot! :)
